test/regression.uts

```diff
@@ -1063,8 +1063,8 @@ str(p)
 assert(_ == '\x80!\x01\x00\x00\x1f\x81\x06\x01\x02\x03\x04\x83\x06\x05\x06\x07\x08{\tABCDEFG\x84\x06\t\n\x0b\x0c')
 PPP(_)
 q=_
-assert( q[PPP_IPCP_Option].type == 123 )
-assert( q[PPP_IPCP_Option].data == 'ABCDEFG' )
+assert( q[PPP_IPCP].options[2].type == 123 )
+assert( q[PPP_IPCP].options[2].data == 'ABCDEFG' )
```

Comment:
I think it would be better to use `q[PPP_IPCP_Option,3]` instead of `q[PPP_IPCP].options[2]` in theses tests, because it is closer to what was implemented before your changes, and also because it shows it is still possible to access the layers you want to access using `PPP_IPCP` and that you don't have to "cheat" as you did.